uc-crux-llvm/src/UCCrux/LLVM/FullType/MemType.hs

```diff
@@ -46,6 +46,7 @@ toMemType =
     FTNonVoidFuncPtrRepr retRepr argsRepr -> funType (Just retRepr) argsRepr
     FTOpaquePtrRepr _ident -> PtrType OpaqueType
     FTArrayRepr natRepr fullTypeRepr -> ArrayType (natValue natRepr) (toMemType fullTypeRepr)
+    FTUnboundedArrayRepr fullTypeRepr -> ArrayType 0 (toMemType fullTypeRepr)
```

Comment:
Is it always going to be a zero here? Is this just the only representation in Crucible-llvm memory types?

Reply:
I'm not totally sure I understand the second part of your question, but yeah, this is how `MemType` represents these unbounded/unsized array types.